definitions/procedures/katello_service/restart.rb

```diff
@@ -0,0 +1,17 @@
+module Procedures::KatelloService
+  class Restart < ForemanMaintain::Procedure
+    metadata do
+      description 'katello-service restart'
+      param :only, 'A comma-separated list of services to include', :array => true
+      param :exclude, 'A comma-separated list of services to skip', :array => true
+    end
+
+    def run
```

Comment:
Currently, the next step describes as:

```
Continue with step [katello-service restart]
```

By overriding the `runtime_description` method, we should be able to get also info about `--only` and `--exclude`
options